t/mojo/request.t

```diff
@@ -49,6 +49,7 @@ is $req->version,           '1.1', 'right version';
 is $req->url,               '/', 'right URL';
 is $req->cookie('a'), undef, 'no value';
 is $req->body, '', 'no content';
+ok !defined $req->headers->content_length, '"Content-Length" does not exist';
```

Comment:
Would it be more robust to do the following..?
```
is int(@{$req->headers->every_header('Content-Length')}), 0, '"Content-Length" does not exist';
```

Reply:
Why?

Reply:
I guess "Why?" is the same as "No" 😄